.pre-commit-config.yaml

```
@@ -0,0 +1,18 @@
+ci:
+  autoupdate_commit_msg: "chore: [pre-commit.ci] pre-commit autoupdate"
+  autoupdate_schedule: "monthly"
```

```suggestion
  autoupdate_schedule: monthly
```

I think these might need to be unquoted (given that I had this quoted up till today and noticed that `pyhf`'s hadn't been updated in over a month). Not 100% sure on this being required, but I know it works if it is unquoted.